crates/router/src/connector/aci/transformers.rs

```diff
@@ -58,7 +58,7 @@ pub enum PaymentDetails {
     AciCard(Box<CardDetails>),
     BankRedirect(Box<BankRedirectionPMData>),
     #[serde(rename = "bank")]
-    Wallet,
+    Wallet(Box<WalletPMData>),
```

Comment:
Why is wallet being renamed as bank?

Reply:
It was present before I made any changes so didn't try to change it. It is working without the rename, I will remove it 👍